src/explore-education-statistics-admin/src/pages/AdminDashboard.tsx

```diff
@@ -1,29 +1,68 @@
-import AdminDashboardApprovedForPublication from '@admin/pages/prototypes/components/AdminDashboardApprovedForPublication';
-import AdminDashboardNeedsWork from '@admin/pages/prototypes/components/AdminDashboardNeedsWork';
-import AdminDashboardReadyForApproval from '@admin/pages/prototypes/components/AdminDashboardReadyForApproval';
+import React, { useContext, useEffect, useState } from 'react';
 import RelatedInformation from '@common/components/RelatedInformation';
 import Tabs from '@common/components/Tabs';
 import TabsSection from '@common/components/TabsSection';
-import React from 'react';
-import { RouteChildrenProps } from 'react-router';
+import { Dictionary } from '@common/types';
 import { LoginContext } from '@admin/components/Login';
-import { Authentication } from '@admin/services/PrototypeLoginService';
-import AdminDashboardPublications from '@admin/components/AdminDashboardPublications';
 import DummyPublicationsData from '@admin/pages/DummyPublicationsData';
-import Accordion from '@common/components/Accordion';
+import { Publication } from '@admin/services/publicationService';
+import AdminDashboardPublicationsTab from '@admin/components/AdminDashboardPublicationsTab';
+import groupBy from 'lodash/groupBy';
 import Link from '../components/Link';
 import Page from '../components/Page';
 
-const BrowseReleasesPage = ({ location }: RouteChildrenProps) => {
+const AdminDashboardPage = () => {
+  const emptyPublicationsArray: Publication[] = [];
+
+  const [myPublications, setMyPublications] = useState(emptyPublicationsArray);
+
+  const [inProgressPublications, setInProgressPublications] = useState(
+    emptyPublicationsArray,
+  );
+
+  const authentication = useContext(LoginContext);
+
+  const loadInitialData = () => {
+    const { user } = authentication;
+    const loggedInUserId = user ? user.id : null;
+
+    const fetchedMyPublications =
+      loggedInUserId === null
+        ? []
+        : DummyPublicationsData.allPublications.filter(
+            _ => _.owner.id === loggedInUserId,
```

Comment:
Wouldn't use `_` as a parameter name as this typically signals that you are not intending to use this parameter. It could also be confused with the convention of using `_` for Lodash (although we shouldn't be using Lodash like this anyway).

The same applies for other lambdas in this file as well.

Reply:
Cool, have put names to them, thanks